### ISSUE TITLE
Crash on install of a specific module

### ISSUE THREAD

**Issue by yvaucher** [camptocamp/odoo-dj#92]
```
  File "/opt/odoo/src/odoo/tools/convert.py", line 707, in _tag_record
    id = self.env(context=rec_context)['ir.model.data']._update(rec_model, self.module, res, rec_id or False, not self.isnoupdate(data_node), noupdate=self.isnoupdate(data_node), mode=self.mode)
  File "/opt/odoo/src/odoo/addons/base/ir/ir_model.py", line 1228, in _update
    record = record.create(values)
  File "/opt/odoo/src/addons/product/models/product_template.py", line 267, in create
    template.create_variant_ids()
  File "/opt/odoo/src/addons/product/models/product_template.py", line 400, in create_variant_ids
    'attribute_value_ids': [(6, 0, variant_ids.ids)]
  File "/opt/odoo/src/addons/product/models/product.py", line 316, in create
    product = super(ProductProduct, self.with_context(create_product_product=True)).create(vals)
  File "/opt/odoo/src/addons/mail/models/mail_thread.py", line 228, in create
    thread = super(MailThread, self).create(values)
  File "/opt/odoo/external-src/odoo-prototype/base_dj/models/base.py", line 265, in create
    return super(Base, self).create(vals)
  File "/opt/odoo/src/odoo/models.py", line 3830, in create
    record = self.browse(self._create(old_vals))
  File "/opt/odoo/src/odoo/models.py", line 3893, in _create
    self.env[parent_model].browse(parent_id).write(parent_vals)
  File "/opt/odoo/src/addons/delivery/models/product_template.py", line 14, in write
    res = super(ProductTemplate, self).write(vals)
  File "/opt/odoo/src/addons/stock/models/product.py", line 501, in write
    return super(ProductTemplate, self).write(vals)
  File "/opt/odoo/src/addons/account/models/product.py", line 45, in write
    res = super(ProductTemplate, self).write(vals)
  File "/opt/odoo/src/addons/product/models/product_template.py", line 288, in write
    res = super(ProductTemplate, self).write(vals)
  File "/opt/odoo/src/addons/mail/models/mail_thread.py", line 274, in write
    result = super(MailThread, self).write(values)
  File "/opt/odoo/external-src/odoo-prototype/base_dj/models/base.py", line 233, in write
    self._dj_handle_special_fields_write(vals)
  File "/opt/odoo/external-src/odoo-prototype/base_dj/models/base.py", line 240, in _dj_handle_special_fields_write
    if vals[fname]:
ParseError: "image_medium" while parsing /opt/odoo/local-src/sf_rma_sale_order/data/product.xml:3, near
<record id="rma_service_product_product" model="product.template">
        <field name="name">RMA Service</field>
        <field name="sale_ok" eval="True"/>
        <field name="type">service</field>
        <field name="uom_id" ref="product.product_uom_unit"/>
        <field name="categ_id" ref="product.product_category_all"/>
    </record>
```

Happened in https://github.com/camptocamp/sensefly_odoo/pull/197

**Comment by simahawk** [comment]
@yvaucher this one is fixed here right? https://github.com/camptocamp/odoo-dj/commit/763aa65942e3adb47f7b2b7bca03b7a697557c00

NOTE: if yes, remember to add `(fixes #XX)` at the end of the commit msg to autoclose the issue ;)

**Comment by yvaucher** [comment]
@simahawk not sure it is completely fixed by 763aa65 . For me the work around works. It's an improvement, but I had some unexplained errors I didn't wanted to look into it further. Then ok let's close this and see if it raise somehow again.
